src/lib/protocols/presentproof/messages/Attachment.ts

```diff
@@ -0,0 +1,36 @@
+import { Expose } from 'class-transformer';
+import { v4 as uuid } from 'uuid';
+import { JsonTransformer } from '../../../utils/JsonTransformer';
+
+interface AttachmentOptions {
+  id?: string;
+  mimeType: string;
+  data: {
+    base64: string;
+  };
+}
+
+export class Attachment {
```

Comment:
Is the Attachment for the present-proof protocol different from the issue-credential protocol? Maybe we could reuse the attachment class and make it a general purpose Attachment decorator per [Aries RFC 0017: Attachments](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0017-attachments/README.md)